docs/data-sources/kps_keypair.md

```diff
@@ -0,0 +1,52 @@
+---
+subcategory: "Data Encryption Workshop (DEW)"
+---
+
+# huaweicloud_kps_keypairs
+
+Use this data source to get a list of keypairs.
+
+## Example Usage
+
+```hcl
+variable "keypair_name" {}
+
+data "huaweicloud_kps_keypairs" "test" {
+  name = var.keypair_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) The region in which to obtain the keypairs. If omitted, the provider-level region will
+  be used.
+
+* `name` - (Optional, String) Specifies the name of the keypair.
+
+* `public_key` - (Optional, String) Specifies the imported OpenSSH-formatted public key.
+
+* `fingerprint` - (Optional, String) Specifies the fingerprint of  the keypair.
```

Comment:
too many blank characters between `of` and `the`.

Reply:
fixed